tweet.py

```diff
@@ -0,0 +1,20 @@
+import  point
+
+#Tweet class  be created that contains (composition) a Point object.
+class Tweet():
+
+    def __init__(self,point,text,source,id_str,lang,created_time):
+
+        self.point=point    #The tweet spatial information
+        self.text=text      #The text
+        self.source=source  #the source
+        self.id_str=id_str  #the id_str
+        self.lang=lang      #the language
+        self.created_time=created_time   #the create time
```

Comment:
Something to think about: Is it better to write a class with many arguments that essential parse an existing data structure or a class that can take said data structure and parse it?  If you are coming from a language that supports multiple constructors, you can achieve something similar using classmethod decorators.